src/org/loklak/api/search/GithubProfileScraper.java

```diff
@@ -149,24 +149,24 @@ public static SusiThought scrapeGithub(String profile) {
 			}
 		}
 		/* If Individual User */
-		if (html.getElementsByAttributeValue("class", "vcard-stat").size() != 0) {
+		if (html.getElementsByAttributeValueContaining("class", "user-profile-nav").size() != 0) {
 
-			String followersUrl = html.getElementsByAttributeValueContaining("class", "vcard-stat").get(0).attr("href");
-			githubProfile.put("followers_url", "https://github.com" + followersUrl);
+			String starredUrl = "https://api.github.com/users/" + profile + "/starred";
+			githubProfile.put("starred_url", starredUrl);
 
-			String followers = html.getElementsByAttributeValueContaining("class", "vcard-stat").get(0).tagName("strong").text();
-			githubProfile.put("followers", followers);
+			int starred = Integer.parseInt(html.getElementsByAttributeValue("class", "Counter").get(1).text());
+			githubProfile.put("starred", starred);
 
-			String starredUrl = html.getElementsByAttributeValueContaining("class", "vcard-stat").get(1).attr("href");
-			githubProfile.put("starred_url", "https://github.com" + starredUrl);
+			String followersUrl = "https://api.github.com/users/" + profile + "/followers";
```

Comment:
Though I like this, the problem with this content is the fact that there are hardcoded URL bases which can easily be refactored to.

`String GITHUB_API_BASE = 'https://api.github.com/users/'`

Similarly the endpoints (of course you can come up with better variable names)
```java
String STARRED_ENDPOINT = "/starred";
String FOLLOWERS_ENDPOINT = "/followers";
```